scio-extra/src/main/scala/com/spotify/scio/extra/hll/zetasketch/ZetaSketchHLL.scala

```diff
@@ -67,6 +67,9 @@ final class ZetaSketchHll[T](arrOpt: Option[Array[Byte]], elemOpt: Option[T] = N
     new ZetaSketchHll[T](Option(nhll.serializeToByteArray()))
   }
 
+  /** @return the byte array representation of the hll */
+  def serializeToByteArray(): Array[Byte] = hll.serializeToByteArray()
```

Comment:
Make it `def internal = hll` so you get access to other methods also? More future proof.

Reply:
`hll` is already a `@transient private lazy val` on line 34

Reply:
I know. Hence a `def` just exposes it.

Reply:
@wayoung opinion? Want this merged?

Reply:
no longer needed since hll is public now.